### ISSUE TITLE
Navigation improvements when performing multiple look-ups

### ISSUE THREAD

**Issue by michelleblood** [wfp/editorial-guide#29]
This relates to all pages which include navigation, such as:
- http://cdn.wfp.org/guides/editorial/reference/wordindex/
- http://cdn.wfp.org/guides/editorial/reference/englishacronyms/

When performing one action (such as navigating to `S` http://cdn.wfp.org/guides/editorial/reference/wordindex/#s) it is straight forward to use. However, when I then need to perform a second action (such as then navigate to `F` http://cdn.wfp.org/guides/editorial/reference/wordindex/#f I need to scroll up to the top of the page again to reach the navigation bar.

This is a common scenario during our use of the Editorial Guide.

Would it be possible to find a more usable solution for this scenario, such as a sticky navigation bar or an arrow to return to the top?

**Comment by jrah** [comment]
Agreed, this makes perfect sense. We've discussed a related issue with the navigation problems on Slack. 

It is possible to modify the existing segmented controls to create this: 
<img width="930" alt="screenshot 2016-07-13 14 20 33" src="https://cloud.githubusercontent.com/assets/1245961/16803140/13c277fe-4905-11e6-9d86-990dffdf7d1f.png">

This will follow the user down the page. It will need some refinement, however. @matthewmorek 

We could probably add a return to top arrow too.

Related to #27

**Comment by jrah** [comment]
When the user clicks on the internal links within the lists it is overlapped by the floating navigation.

Clicking on any internal link needs to be offset by the navigation's presence.
